src/db/addTermMgmt.sql

```diff
@@ -346,6 +346,40 @@ alpha_GB_Instructor, alpha_GB_Student, alpha_GB_Registrar, alpha_GB_RegistrarAdm
 alpha_GB_Admissions, alpha_GB_DBAdmin;
 
 
+--Returns a table of rows from the Course table. Without arguments, returns all
+--courses. Returns courses held during the specified year. Returns no rows if a
+--match is not made.
+CREATE OR REPLACE FUNCTION showCoursesByYear()
+RETURNS TABLE(Number VARCHAR(8),
+              Title VARCHAR(100),
+              InstructorFullName VARCHAR(100),
+              StartDate DATE,
+              EndDate DATE
+             )
+AS
+$$
+SELECT number, title, COALESCE(getInstructorName(instructor1),' ') ||
+                      COALESCE(', ' getInstructorName(instructor2),' ') ||
```

Comment:
There seems to be a missing concatenation operator between `', '` and `getInstructorName...` Seems to also occur elsewhere in the file.

Reply:
Thanks for the catch.  Fixed.